test/test_mps.py

```diff
@@ -6985,18 +9711,107 @@ def test_output_match(self, device, dtype, op):
                 cpu_out = op(*cpu_args, **cpu_kwargs)
                 mps_out = op(*mps_args, **mps_kwargs)
                 self.assertEqual(cpu_out, mps_out)
-        except Exception as e:
-            if not generate_new_truth:
-                raise e
-        else:
```

Comment:
Note that this is not in the except, this is in the else that we append to the allow list.

Reply:
Yup, I later realized that too lol